README.md

```diff
@@ -268,10 +268,9 @@ Querying for transactions via `txs` endpoint supports 3 kinds of parameters spec
 - generic ids
 - transaction fields
 
-Pagination supported via specifying of 2 parameters:
-
-- limit
-- page
+To be able to traverse through the list of transactions, the `next` field
+provided on the current page response is provided as explained in the
```

Comment:
"the `next` field provided on the current page response is provided"... 
suggest ->
"the `next` field returned on the current page response is provided"...

Reply:
done